### ISSUE TITLE
Nuxt.js

### ISSUE THREAD

**Issue by bondzula** [e2o/vue-element-query#1]
Hi, i was trying to use this with Nuxt.js however I'm getting:
 ```  
error  in ./node_modules/vue-element-query/src/index.js

Module parse failed: Unexpected token (29:16)
You may need an appropriate loader to handle this file type.
|             const instance = Object.keys(this.eq.breakpoints).reduce(
|               (accumulator, currentValue) => ({
|                 ...accumulator,
|                 [currentValue]: this.$_elementQueryMixin_checkAllConditions(
|                   this.eq.breakpoints[currentValue]

 @ ./plugins/element-query.js 2:0-48
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?name=client&reload=true&timeout=30000&path=/__webpack_hmr ./.nuxt/client.js 
```

What i done is: `plugins/element-query.js` 
```js
import Vue from 'vue';
import VueElementQuery from 'vue-element-query';

Vue.use(VueElementQuery); 
```

And in `nuxt.config.js`:
```js
...
plugins: ['~/plugins/element-query']
...
```

**Comment by e2o** [comment]
@bondzula Fixed in `vue-element-query@1.0.6`, thanks for bringing this to my attention. Feel free to comment if not resolved.

**Comment by bondzula** [comment]
It's fixed, thanks a lot!!!!
